db/migrate/20181023175940_add_column_to_category.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToCategory < ActiveRecord::Migration[5.2]
```

Comment:
I think the title of the migration is supposed to reflect the name of the column, eg `AddStatusToCategory`